### ISSUE TITLE
Suggestion for CreateSprite for words in container rather than just chars

### ISSUE THREAD

**Issue by shanemccartney** [Gamua/Starling-Framework#724]
I find its useful to deal with chars in context to words this revision allows for a change where by you can group and animate words of chars.

http://pastie.org/10236476

**Comment by PrimaryFeather** [comment]
Good idea, you're right! I'll consider supporting that (perhaps optionally, with an additional parameter).
